class.media-summary.php

```diff
@@ -252,6 +252,14 @@ static function get_excerpt( $post_content, $post_excerpt, $max_words = 16, $max
 				'read_more_threshold' => 25,
 			) ) );
 		} else {
+			
+			/**
+			 * Filter the post excerpt.
+			 *
+			 * @since 2.6.0
+			 *
+			 * @param str $post_excerpt The post excerpt.
+			 */
```

Comment:
This filter comes from Core I think:
https://github.com/WordPress/WordPress/blob/dfaa89d8531a3e34ec6c19b72f01e95ea3527fa0/wp-includes/post-template.php#L366